examples/StandardFirmataWiFi/StandardFirmataWiFi.ino

```diff
@@ -871,6 +881,7 @@ void printWifiStatus() {
     DEBUG_PRINT( "WiFi connection failed. Status value: " );
     DEBUG_PRINTLN( WiFi.status() );
   }
+#ifdef SERIAL_DEBUG
```

Comment:
What is this doing for us? How does `SERIAL_DEBUG` get defined?

Reply:
SERIAL_DEBUG can be found in StandardFirmataWifi.ino:90. If not enabled with ESP32 SDK (default) the compiler will raise a warning or an error (depending on the compiler settings) because of the unused local variables like "rssi" in line 898, caused by defining DEBUG_PRINT to nothing (see utility/firmataDebug.h).